instructors.md

```diff
@@ -337,6 +228,132 @@ a substantial fraction won't be able to navigate to the right directory without
+    the audio cross-talk makes the recordings unintelligible.
+
+*   We use Etherpad for in-person training,
+    both for note-taking and for posting exercise solutions and feedback on recorded lessons
+    Questions and discussion are done aloud.
+
+### Two-Day Online With Groups
+
+*   We use Google Hangouts and Etherpad as in the multi-week version.
+    Each group of learners is together in a room using one camera and microphone,
+    rather than each being on the call separately.
+    We have found that having good audio matters more than having good video,
+    and that the better the audio,
+    the more learners can communicate with the instructor and other rooms by voice
+    rather than by using the Etherpad chat.
+
+*   We do the video lecture exercise as in the two-day in-person training.  
+
+<a name="schedule"></a>
+## Schedule
+
+The schedule for the course is evolving along with the content,
+but here's what we do in a typical two-day event:
+
+| Day 1 | 09:00 | Welcome: key concepts and training goals            |
+|       | 09:30 | Novice mind-set; formative vs. summative assessment |
+|       |       | *Exercise: multiple choice questions*               |
+|       | 10:30 | Coffee                                              |
+|       | 10:45 | Lesson study; giving feedback                       |
+|       |       | *Exercise: video recorded teaching*                 |
+|       | 11:45 | Morning wrap-up (minute cards)                      |
+|       | 12:00 | Lunch                                               |
+|       | 13:00 | Concept maps                                        |
+|       |       | *Exercise: draw and critique concept maps*          |
+|       | 14:15 | Coffee                                              |
+|       | 14:30 | Learning objectives                                 |
+|       |       | *Exercise: write and critique learning objectives*  |
+|       | 15:15 | Cognitive load                                      |
+|       | 15:30 | *Exercise: faded example*                           |
+|       | 15:45 | Wrap-up (one up, one down)                          |
+|       | 16:00 | Close                                               |
+|       |       | *Homework: read operations guide*                   |
+| Day 2 | 09:00 | Live coding demonstration                           |
+|       |       | *Exercise: live coding practice*                    |
+|       | 10:15 | Coffee                                              |
+|       | 10:30 | Review questions about operations guide             |
+|       | 11:15 | *Exercise: create workshop website*                 |
+|       | 11:45 | Wrap-up (minute cards)                              |
+|       | 12:00 | Lunch                                               |
+|       | 13:00 | Data Carpentry                                      |
+|       | 13:30 | Motivators and demotivators                         |
```

Comment:
From my experience this takes about 10-15 minutes to talk about (once you start dwelling on this, people loose interest). Their own stories - most people are done in 15-20 min. So could some of this left out time for the infrastructure of the Carpentries. (see my comment below)